src/extensions/bigbang/test/bigbang_test.go

```diff
@@ -103,9 +116,16 @@ func testConnection(t *testing.T) {
 	require.Equal(t, 200, resp.StatusCode)
 }
 
-func zarfExec(t *testing.T, args ...string) {
-	err := exec.CmdWithPrint(zarf, args...)
-	require.NoError(t, err)
+func zarfExec(t *testing.T, args ...string) (string, string, error) {
```

Comment:
![image](https://github.com/defenseunicorns/zarf/assets/3977569/22c7b6fa-aaa5-44af-aa59-06d2665221c9)
lint, also we should still check `require.NoError(t, err)` where applicable rather than dogsledding in the caller.

Reply:
Removed unused `t` parameter from `zarfExec` and replaced all dogsleds with error handling